src/NoTimeToWaitBundle/Resources/views/Default/nttw5.html.twig

```diff
@@ -1,20 +1,23 @@
 {% extends 'NoTimeToWaitBundle::base.html.twig' %}
 
-{% block title %}No Time to Wait 5: Wait for It!{% endblock %}
+{% block title %}No Time to Wait 5: Open Isn't Enough!{% endblock %}
 
 {% block body %}
 <h1><strong>No Time to Wait!</strong> *</h1>
 
 <h2>The No Time to Wait Conference</h2>
 <p><a href="https://www.beeldengeluid.nl/">Netherlands Institute for Sound and Vision</a> shall host the fifth No Time to Wait conference. No Time to Wait is a FREE three-day conference focused on <em>open media</em>, <em>open standards</em>, and <em>digital audiovisual preservation</em> managed by <a href="{{ path('homepage') }}">MediaArea.net</a>. No Time to Wait has always been a conference that blends in-person and online participation and this year we shall work to emphasize online participation in response to the social, travel, and financial restrictions related to the pandemic.</p>
-
+<p>The theme this year is "Open Isn't Enough" highlighting that simply bringing openness into our work is not enough to propogate its adoption and  accessibility, but that open media and open standards require an environment of skill-sharing, advocacy, collaboration, and community-building to be truly useful.</p>
 <h2>When</h2>
 <p>December 8-10, 2021</p>
 <p><i>Note: Due to the ongoing coronavirus (COVID-19) pandemic, No Time To Wait, which was originally scheduled for September 2020, had a series of postponements. As scientific and social progress continues to address the pandemic, we continue to follow policy developments and are hopeful that we can hold a safe and successful conference.</i></p>
 <p>Call for proposals and conference registration coming soon.</p>
 
-<h2>Survey</h2>
-<p>We are currently developing a timeline to announce a call for proposals and open registration soon. To help us prepare for this, we request that you complete <a href="http://bit.ly/nttw5prelimsurvey">our preliminary survey</a>.</p>
+<h2>Registration</h2>
+<p>If you'd like to participate in No Time to Wait 5, either in-person or online, please complete <a href="https://bit.ly/nttw5signup">our reigstration form</a>. Note that space is limited.</p>
```

Comment:
"reigstration" typo.
Maybe "in-person space is limited" as online space is not (at least far from how many online people we expect)